AutomatedTesting/Gem/PythonTests/editor/EditorScripts/ComponentCRUD_Add_Delete_Components.py

```diff
@@ -100,7 +101,7 @@ async def add_component(component_name):
         general.select_object("Entity1")
 
         # Give the Entity Inspector time to fully create its contents
-        general.idle_wait(0.5)
+        wait_for_propagation()
```

Comment:
I don't think there's anything before this call has changes that need to be propagated to the prefab instance. This should just be general.idle_wait_frames(n), where n is the number of frames it takes for inspector to be ready.

Reply:
Fixed.